docs/documentation/getting-started.md

```diff
@@ -98,193 +34,52 @@ accompany your sessions.
-- transfer the initial content of the module shared by the host to all participating clients
-- work on shared resources
-- create, delete, and move resources in the shared module
-- interact freely with non-shared resources
-- follow other participants of the session ([follow mode](features.md#follow-mode))
-
+Most interactions with the Saros session logic (like starting or ending a session) can be done through the Saros tool window.
+This window is attached to the bottom right of the IDE by default.
+It is marked with the title "Saros" and the Saros icon (![saros icon](images/icons-i/saros.png)).
+All actions described in the following sections take place in this tool window.
 
-### Sharing Complex Modules
+{% endcapture %}
+{% include ide-tabs.html eclipse=eclipse intellij=intellij %}
 
-Even though Saros offers the option to create the module on the client side as part of the session, this should only be used for relatively simple modules.
-For more complex modules, it is advised to share the module structure some other way (e.g. a VCS) before starting a session. Saros currently does not set up things like libraries or module dependencies, meaning they would have to be configured by hand.
+## Start a Session
 
-## Compatibility
+In the following we show you how to set up Saros to start your first session.
 
-The current release `0.2.2` is not compatible with the previous Saros/I releases (`0.1.0`, `0.2.0`, or `0.2.1`) or other Saros plugins (like Saros/E).
+### Step 1: Creating an XMPP Account
 
+You first need an **XMPP account** to use Saros.
 
-## How to Use Saros/I
+1.  You can **create a new account**:
+  * On our XMPP server using our **[signup page](https://saros-con.imp.fu-berlin.de:5280/register/new)**
+        * Note: Due to abuse by spammers, we had to disable in-band registration and from within Saros.
+  * On other XMPP servers as [jabber.org](https://www.jabber.org/) ([register.jabber.org](https://register.jabber.org/)).
+2.  You can **use an existing account**
+3.  You [setup your own XMPP server](setup-xmpp.md).
 
-Most interactions with the Saros session logic (like starting or ending a session) can be done through the Saros tool window.
-This window is attached to the bottom right of the IDE by default.
-It is marked with the title "Saros" and the Saros icon (![saros icon](images/icons-i/saros.png)).
-All actions described in the following sections take place in this tool window.
+* Please note that your performance may vary depending on the XMPP server settings, such as bandwidth restrictions or feature supports.
+* It is recommended to **use the same XMPP server** for all users
+which participate in a Saros session. Our tests have shown
+that many public XMPP servers use outdated software
+and certificates. Thus in many cases communication between federated
+servers will fail.
 
-### Creating an XMPP Account
+### Step 2: Adding Account and Connecting
```

Comment:
A section on modifying/deleting added XMPP accounts (and the lack of the feature for Saros/I) is missing.